### DISCUSSION TITLE
Full screen notifications

### DISCUSSION THREAD

**Discussion by macgyverkam** [pakerwreah/Calendr#248] [Ideas]
I recently found an app that allows for displaying full-screen notifications. Maybe such a feature would also be useful in Calendr.
![Xnip2024-09-23_11-34-25](https://github.com/user-attachments/assets/539d1190-ba4b-49a0-ab73-d89fbc3abd53)

https://apps.apple.com/us/app/dato/id1470584107?mt=12

**Comment by pakerwreah** [comment]
I understand that this may be useful for some people, but it's beyond the scope of Calendr.

People have asked me for flashing lights and sound, but I always say no, because it's not meant to replace any notification system, just be available in the menu bar for a quick look.

Even if I implement this, I won't be using it or testing it, which creates another problem.

**Comment by pakerwreah** [comment]
Closing this in favor of https://github.com/pakerwreah/Calendr/discussions/283
